contrib/mesos/pkg/executor/executor.go

```diff
@@ -388,53 +380,21 @@ func (k *Executor) LaunchTask(driver bindings.ExecutorDriver, taskInfo *mesos.Ta
 	// (a) we ignore future launches for it
 	// (b) we have a record of it so that we can kill it if needed
 	// (c) we're leaving podName == "" for now, indicates we don't need to delete containers
+	var launchTimer *time.Timer
+	if k.launchGracePeriod > 0 {
+		launchTimer = time.NewTimer(k.launchGracePeriod)
+	} else {
+		launchTimer = &time.Timer{}
```

Comment:
I don't want a timer that has an expired C chan, I want a nil C (one that will always block). This logic isn't very complicated. I can add docs to clarify the intent